server/graphql/v1/mutations/expenses.js

```diff
@@ -671,6 +672,22 @@ export async function payExpense(req, args) {
     );
   }
 
+  // If the amount of the expense is larger than i.e. $1000 enforce 2FA if the user has it turned on
+  const payoutMethodsToEnforceTwoFactorAuthenticationOn = [
```

Comment:
This would be a good variable name for the array itself, but here I think we can simplify as `is2FARequiredForPayoutMethod`